### PR TITLE
Adds Abductors and Flypeople as valid Halloween races!

### PR DESCRIPTION
**Pull request by fluffe9911**
None of this should be that overpowered or cause to much of a headache so why not add more variety for this Halloween season!

### CODE REVIEW

**Comment by GuppyLaxx** [comment]
synths are broken as fuck, remove them

**Comment by Ktlwjec** [comment]
i will council approve flypeople

**Comment by MegaEmpirical** [comment]
I wish flypeople were always available. I want to eat vomit,

File: code/modules/mob/living/carbon/human/species_types/abductors.dm [hunk 1]
```diff
@@ -8,6 +8,11 @@
 	mutanttongue = /obj/item/organ/tongue/abductor
 	var/scientist = FALSE // vars to not pollute spieces list with castes
 
+/datum/species/abductor/check_roundstart_eligible()
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/species_types/abductors.dm]: Modular

File: code/modules/mob/living/carbon/human/species_types/angel.dm [hunk 2]
```diff
@@ -141,4 +141,4 @@
 		H.movement_type &= ~FLYING
 		override_float = FALSE
 		H.pass_flags &= ~PASSTABLE
-		H.CloseWings()
\ No newline at end of file
+		H.CloseWings()
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/species_types/angel.dm:144]: Delete this change

File: code/modules/mob/living/carbon/human/species_types/flypeople.dm [hunk 3]
```diff
@@ -10,6 +10,11 @@
 	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/fly
 	disliked_food = null
 	liked_food = GROSS
+	
+/datum/species/fly/check_roundstart_eligible()
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/species_types/flypeople.dm]: Modular

File: code/modules/mob/living/carbon/human/species_types/synths.dm [hunk 4]
```diff
@@ -126,4 +126,4 @@
 		else
 			return ..()
 	else
-		return ..()
\ No newline at end of file
+		return ..()
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/species_types/synths.dm:129]: Delete this change

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/angel.dm (hunk 2 above)
Comment (fluffe9911) [reply code/modules/mob/living/carbon/human/species_types/angel.dm:144]: Dont know how to fix my computer is broken so been using the ingithub stuff and everytime I remove the space it readds it

File: code/modules/mob/living/carbon/human/species_types/synths.dm (hunk 4 above)
Comment (fluffe9911) [reply code/modules/mob/living/carbon/human/species_types/synths.dm:129]: Dont know how to fix my computer is broken so been using the ingithub stuff and everytime I remove the space it readds it

Comment (fluffe9911) [reply code/modules/mob/living/carbon/human/species_types/synths.dm:129]: but yeah if your able to can you fix it for me? if not just close this pr and I will open a new one that does not mess with the angel and synth files.

File: code/modules/mob/living/carbon/human/species_types/angel.dm (hunk 2 above)
Comment (fluffe9911) [reply code/modules/mob/living/carbon/human/species_types/angel.dm:144]: but yeah if your able to can you fix it for me? if not just close this pr and I will open a new one that does not mess with the angel and synth files.

File: code/modules/mob/living/carbon/human/species_types/flypeople.dm [hunk 5]
```diff
@@ -10,6 +10,11 @@
 	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/fly
 	disliked_food = null
 	liked_food = GROSS
+	
+/datum/species/fly/check_roundstart_eligible() //YogsTG Change
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/species_types/flypeople.dm]: That's not how you modularize something

File: code/modules/mob/living/carbon/human/species_types/abductors.dm [hunk 6]
```diff
@@ -8,6 +8,11 @@
 	mutanttongue = /obj/item/organ/tongue/abductor
 	var/scientist = FALSE // vars to not pollute spieces list with castes
 
+/datum/species/abductor/check_roundstart_eligible() //YogsTG Change
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/species_types/abductors.dm]: That's not how you modularize something

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Review by ThatLing — CHANGES_REQUESTED** [review]
Abductors, no; Flypeople, yes.

**Comment by fluffe9911** [comment]
Heck making it a halloween race then may try to just make fly people a 24/7 race in the future

**Pull request closed without merging**